README.md

```diff
@@ -14,54 +14,42 @@ for an overview of features.
 
 ## Getting Started
 
-### Downloading
-
-You can install the Git LFS client in several different ways, depending on your
-setup and preferences.
-
-* **Linux users**. Debian and RPM packages are available from
-  [PackageCloud](https://packagecloud.io/github/git-lfs/install).
-* **macOS users**. [Homebrew](https://brew.sh) bottles are distributed, and can
-  be installed via `brew install git-lfs`.
-* **Windows users**. Git LFS is included in the distribution of
-  [Git for Windows](https://gitforwindows.org/). Alternatively, you can
-  install a recent version of Git LFS from the [Chocolatey](https://chocolatey.org/) package manager.
-* **Binary packages**. In addition, [binary packages](https://github.com/git-lfs/git-lfs/releases) are
-available for Linux, macOS, Windows, and FreeBSD.
-* **Building from source**. [This repository](https://github.com/git-lfs/git-lfs.git) can also be
-built from source using the latest version of [Go](https://golang.org), and the
-available instructions in our
-[Wiki](https://github.com/git-lfs/git-lfs/wiki/Installation#source).
+### Installing
 
-Note that Debian and RPM packages are built for all OSes for amd64 and i386.
-For arm64, only Debian packages for the latest Debian release are built due to the cost of building in emulation.
+#### On Linux
 
-### Installing
+Debian and RPM packages are available from packagecloud, see [installation instructions](INSTALLING.md).
+
+#### On macOS
+
+[Homebrew](https://brew.sh) bottles are distributed and can be installed via `brew install git-lfs`.
+
+#### On Windows
+
+Git LFS is included in the distribution of [Git for Windows](https://gitforwindows.org/).
+Alternatively, you can install a recent version of Git LFS from the [Chocolatey](https://chocolatey.org/) package manager.
 
 #### From binary
 
-The [binary packages](https://github.com/git-lfs/git-lfs/releases) include a script which will:
+[Binary packages](https://github.com/git-lfs/git-lfs/releases) are
+available for Linux, macOS, Windows, and FreeBSD.
+The binary packages include a script which will:
 
 - Install Git LFS binaries onto the system `$PATH`
-- Run `git lfs install` to
-perform required global configuration changes.
+- Run `git lfs install` to perform required global configuration changes.
 
 ```ShellSession
 $ ./install.sh
 ```
 
-#### From source
+Note that Debian and RPM packages are built for all OSes for amd64 and i386.
+For arm64, only Debian packages for the latest Debian release are built due to the cost of building in emulation.
 
-- Ensure you have the latest version of Go, GNU make, and a standard Unix-compatible build environment installed.
-- On Windows, install `goversioninfo` with `go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@latest`.
-- Run `make`.
-- Place the `git-lfs` binary, which can be found in `bin`, on your system’s executable `$PATH` or equivalent.
-- Git LFS requires global configuration changes once per-machine. This can be done by
-running:
```

Comment:
Hi @chrisd8088 Thank you for your review and feedback! 🙇‍♂️ ❤️ 

I found the wiki with the Installation instructions only at the end of performing my pull request changes (since it was only linked to from the "From source" section).

Now I have to ask, why are there installation instructions in the README _and_ in the wiki? 😊
From the user perspective I find it quite confusing as you don't know which is the most up-to-date and the source of truth, from the maintainer perspective I also would find it as an unnecessary overhead?

Wouldn't it make sense to consolidate these instructions in one place? 🤔
(I'm happy to help with that.)

The overall goal would be to make it as easy and straight forward for users to install Git LFS and reduce confusion and friction.

Reply:
I would assume that different folks created the two sets of instructions at different times.  I think it's probably advisable for the Wiki to refer to the canonical instructions in the repo's `README.md` rather than the other way around.

Reply:
> I think it's probably advisable for the Wiki to refer to the canonical instructions in the repo's README.md rather than the other way around.

I agree! We can then link to the README from the wiki.

If it's 👍 for you I would work then on moving and consolidating the installation instructions in the README and update this PR accordingly.

Reply:
Well, I was only talking about the Wiki's copy of these particular notes, not the whole `INSTALLING.md` page, if that was what you were thinking.  If you don't mind restoring these notes in the `README.md`, that would get my approval -- thank you!

Reply:
> If you don't mind restoring these notes in the README.md, that would get my approval -- thank you!

Hi @chrisd8088,

In https://github.com/git-lfs/git-lfs/pull/5260/commits/6b2aa30be924e8b2835bc380186453259a39c4d6 I have restored the "From source" section from the original `README.md`.
The only change I made is making `git lfs install` inline as suggested in https://github.com/git-lfs/git-lfs/pull/5260#discussion_r1084825389.

For convenience here is the updated rendered `README.md`:
https://github.com/steffen/git-lfs/blob/main/README.md

I think these changes should make the PR approvable.
As discussed, further potential improvements I will propose in separate bitesized pull requests over time.

Thank you again for your review and feedback! 🙇‍♂️

Reply:
Looks great, thank you!